docs/tutorial/quotes.md

```diff
@@ -296,36 +296,37 @@ We also used the `Varargs` extractor to match the arguments of type `Expr[Seq[St
 Then we used the `Consts` to match known constants in the `Seq[Expr[String]]` to get a `Seq[String]`.
 
 
-## The QuoteContext
-The `QuoteContext` is the main entry point for the creation of all quotes.
+## The Quotes
+The `Quotes` is the main entry point for the creation of all quotes.
 This context is usually just passed around through contextual abstractions (`using` and `?=>`).
-Each quote scope will provide have its own `QuoteContext`.
+Each quote scope will provide have its own `Quotes`.
 New scopes are introduced each time a splice is introduced `${...}`.
-Though it looks like a splice takes an expression as argument, it actually takes a `QuoteContext ?=> Expr[T]`.
+Though it looks like a splice takes an expression as argument, it actually takes a `Quotes ?=> Expr[T]`.
 Therefore we could actually write it explicitly as `${ (using q) => ... }`, this might be useful when debugging to avoid generated names for these scopes.
 
-The method `scala.quoted.qctx` provides a simple way to use the current `QuoteContext` without naming it.
-It is usually imported along with the `QuoteContext` using `import scala.quoted._`.
+The method `scala.quoted.quotes` provides a simple way to use the current `Quotes` without naming it.
+It is usually imported along with the `Quotes` using `import scala.quoted._`.
 
 ```scala
 ${ (using q1) => body(using q1) }
 // equivalent to
-${ body(using qctx) }
+${ body(using quotes) }
 ```
-If you explicitly name a `QuoteContext` `qctx` you will shadow this definition.
+If you explicitly name a `Quotes` `quotes` you will shadow this definition.
 
 When we write a top level splice in a macro we are calling something similar to the following definition.
-This splice will provide the initial `QuoteContext` associated with the macro expansion.
+This splice will provide the initial `Quotes` associated with the macro expansion.
 ```scala
-def $[T](x: QuoteContext ?=> Expr[T]): T = ...
+def $[T](x: Quotes ?=> Expr[T]): T = ...
 ```
 
 When we have a splice within a quote, the inner quote context will depend on the outer one.
-This link is represented using the `QuoteContext.Nested` type.
-This relation tells us that it is safe to use expressions created with `q1` within the scope of `q2` but not the other way around (this constraint is statically checked yet).
+This link is represented using the `Quotes.Nested` type.
+Users of this quotes will almost never need to use `Quotes.Nested`.
+These details are only useful for advanced macros that will inspect code and may encounted quotes and splices.
```

Comment:
```suggestion
This link is represented using the `Quotes.Nested` type.
Users of quotes will almost never need to use `Quotes.Nested`.
These details are only useful for advanced macros that will inspect code and may encounter details of quotes and splices.
```
I don't understand the last part of line 326 containing "encounted". I changed it to "encounter", is that right?